person.rb

```diff
@@ -9,9 +7,10 @@ def initialize(age, name, parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
   end
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rental
```

Comment:
- [ ] Since you pluralized `rentals` in the Rental class, kindly do the same here.

  ```
  :rentals
  @rentals = [ ]
  ```